extensions/interactions/RatioExpressionInput/RatioExpressionInput.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, softwar
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Python configuration for RatioExpressionInput interaction."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from extensions.interactions import base
+
+
+class RatioExpressionInput(base.BaseInteraction):
+    """Interaction for ratio input."""
+
+    name = 'Ratio Input'
+    description = 'Allows learners to enter integers in ratio format.'
+    display_mode = base.DISPLAY_MODE_INLINE
+    is_trainable = False
+    _dependency_ids = []
+    answer_type = 'RatioExpression'
+    instructions = None
+    narrow_instructions = None
+    needs_summary = False
+    can_have_solution = True
+    show_generic_submit_button = True
+
+    _customization_arg_specs = [{
+        'name': 'customPlaceholder',
+        'description': 'Custom placeholder text (optional)',
+        'schema': {
+            'type': 'custom',
+            'obj_type': 'SubtitledUnicode'
+        },
+        'default_value': {
+            'content_id': None,
+            'unicode_str': ''
+        }
+    }, {
+        'name': 'noOfTerms',
+        'description': 'Minimum number of terms required by the learner\'s.',
```

Comment:
I already mentioned this in the design doc, but you should explain to the creator what 0 means.

Also, this is not the minimum number of terms -- if specified, it should be the actual number of terms. It should not be allowed to equal 1 (enforce this using validation errors).

You can use the following explanation:

```
The number of elements that the answer must have. If set to 0, a ratio of any length will be accepted.
```